### PR TITLE
return this for destroy for consistency

### PR DESCRIPTION
**Pull request by ahumphreys87**

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ _.extend(Marionette.Controller.prototype, Backbone.Events, {
 
     this.stopListening();
     this.off();
+
```
Comment (samccone) [inline src/controller.js]: :x: drop change

**Comment by samccone** [comment]
We will want to update the docs, and the .yaml stubs

**Comment by ahumphreys87** [comment]
@samccone updated docs and yaml

**Comment by samccone** [comment]
:clap: 

:+1: from me

ping @marionettejs/marionette-core

**Comment by cmaher** [comment]
:+1:

**Pull request merged**